work_queue/src/batch_job_sandbox.c

```diff
@@ -0,0 +1,292 @@
+#include "batch_job.h"
+#include "batch_job_internal.h"
+#include "debug.h"
+#include "process.h"
+#include "macros.h"
+#include "stringtools.h"
+#include "unlink_recursive.h"
+
+#include <time.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+#include <signal.h>
+#include <sys/stat.h>
+#include <dirent.h>
+
+#define CHAR_ARR_LEN 4096
+
+static struct itable *oup_fn_table = NULL;
```

Comment:
Other people will read this code, so spell these out so that the purpose is obvious: `output_file_table` and `output_sandbox_table` or similar.